### PR TITLE
Fixing scoped build

### PR DESCRIPTION
**Pull request by samsonasik**

### CODE REVIEW

File: .github/workflows/build_scoped_rector.yaml [hunk 1]
```diff
@@ -43,7 +43,10 @@ jobs:
                     COMPOSER_TOKEN: ${{ secrets.ACCESS_TOKEN }}
 
             # install only prod dependencies - do not use ramsey, it uses cache including "dev", we want to avoid it here
-            -   run: composer install --no-dev --ansi
+            # somehow psr/http-message needed now when downgrading ssch/typo3-rector
+            -   run: |
+                    composer require psr/http-message
+                    composer update --no-dev --ansi
```
Comment (samsonasik) [inline .github/workflows/build_scoped_rector.yaml:49]: `psr/http-message` somehow now required when downgrading ssch/typo3-rector

File: build/config/config-downgrade.php [hunk 2]
```diff
@@ -64,6 +63,7 @@
                 \Symplify\RuleDocGenerator\Contract\Category\CategoryInfererInterface::class,
                 \PhpParser\PrettyPrinterAbstract::class,
                 \Helmich\TypoScriptParser\Parser\Traverser\Visitor::class,
+                \Symplify\SymplifyKernel\Contract\LightKernelInterface::class,
```
Comment (samsonasik) [inline build/config/config-downgrade.php:66]: somehow overlapped with `RectorKernel`, so register to typed safe for DowngradeParameterTypeWideningRector

File: utils/compiler/src/Command/DowngradePathsCommand.php [hunk 3]
```diff
@@ -44,8 +44,10 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             'vendor/symfony/dependency-injection',
             'vendor/symfony/console',
             'vendor/symfony vendor/psr',
-            'vendor/symplify vendor/nikic vendor/ssch bin src packages rector.php',
+            'vendor/symplify vendor/nikic src packages',
+            'vendor/ssch',
```
Comment (samsonasik) [inline utils/compiler/src/Command/DowngradePathsCommand.php:48]: `vendor/ssch` is part that take longest time to downgrade, so split to separate process.

**Comment by samsonasik** [comment]
@TomasVotruba the scoped build is now pushed to rector/rector, the `e2e/define-constant` failing test now displayed at 

https://github.com/rectorphp/rector/runs/4106986477?check_suite_focus=true#step:5:7


for PHP 7.x

File: .github/workflows/build_scoped_rector.yaml (hunk 1 above)
Comment (TomasVotruba) [reply .github/workflows/build_scoped_rector.yaml:49]: Interesting... This should be probably solved in the typo3 Rector package.

@sabbelasichon Could you check it and remove the package if possible?

**Pull request merged**